api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -567,6 +567,8 @@ public static final Collection<String> AUTO_ROLES() {
 	public static final String AUTO_CLOSE_VISITS_TASK_NAME = "Auto Close Visits Task";
 	
 	public static final String GP_ALLOWED_FAILED_LOGINS_BEFORE_LOCKOUT = "security.allowedFailedLoginsBeforeLockout";
+
+	public static final String 	GP_WAITING_TIME_UNLOCK_ACCOUNT_AFTER_LOCKOUT = "security.waitingTimeUnlockAccountAfterLockout";
```

Comment:
What do you think of `security.unlockAccountWaitingTime` and GP_UNLOCK_ACCOUNT_WAITING_TIME?

Reply:
We've made the change.